lib/Dependency.js

```diff
@@ -71,6 +72,10 @@ class Dependency {
 		return null;
 	}
 
+	/**
+	 * @param {Hash} hash - hash
```

Comment:
```suggestion
	 * @param {Hash} hash hash
```

Don't put dashes here.

Reply:
https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-require-hyphen-before-param-description

Reply:
done